test/unit/capsize_deployer_test.rb

```diff
@@ -193,17 +193,17 @@ def test_handling_of_prompt_configuration
     assert stage_with_prompt.deployment_possible?, stage_with_prompt.deployment_problems.inspect
 
     # add a config value that wants a promp
-    stage_with_prompt.configuration_parameters.build(:name => 'password', :prompt_on_deploy => 1).save!
+    stage_with_prompt.configuration_parameters.create(:name => 'password', :prompt_on_deploy => 1)
     assert !stage_with_prompt.prompt_configurations.empty?
 
     # create the deployment
-    deployment = create_new_deployment(:stage => stage_with_prompt, :task => 'deploy', :prompt_config => {:password => '123'})
+    deployment = create_new_deployment(:stage => stage_with_prompt, :task => 'deploy', :prompt_config => {'password' => 'pass1234'})
 
     write_deployer_files(deployment)
-    file = File.open(rooted("#{stage_with_prompt.project.capsize_project_name}/#{stage_with_prompt.name}.rb"))
+    file = File.open(rooted("#{@project.capsize_project_name}/#{stage_with_prompt.name}.rb"))
     contents = file.read
 
-    assert_match(/set :password, "123"/, contents)
+    assert_match(/set :password, 'pass1234'/, contents)
```

Comment:
This looks extremely suspicious. It's worth a comment in the code to describe what's going on — you have a hardcoded password?